src/aznfswatchdog

```diff
@@ -607,10 +607,10 @@ while :; do
             if ! update_mountmap_entry "$line" "$l_host $l_ip $new_ip"; then
                 eecho "Will reattempt the operation in next iteration."
             else
-                mountpoint=$(echo "$findmnt" | grep " nfs ${l_ip}:" | awk '{print $4}')
-                mountpoint=$(echo -e "$mountpoint")
-                if [ -n "$mountpoint" ]; then
-                    ping_new_endpoint "$mountpoint" &
+                mountpoint2048=$(echo "$findmnt" | egrep " nfs ${l_ip}:.*\<port=2048\>" | awk '{print $4}' | head -n 1)
```

Comment:
you can omit the head -n1 by using -m1 option to egrep

mountpoint2048=$(echo "$findmnt" | egrep -m1 " nfs ${l_ip}:.*\<port=2048\>" | awk '{print $4}')